src/test/java/componenttest/FeatureTest.java

```diff
@@ -0,0 +1,23 @@
+package componenttest;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class FeatureTest extends AbstractComponentTest {
+
+    // Just an example
+    @Test
+    public void featureref_returns_some_features() throws Exception {
```

Comment:
I totally agree with you if it comes to production code, but in tests (especially in component tests) I like the method names to as descriptive as possible. So I try to write whole sentence as the method name and it is better readable with underlines than in camel case.